src/components/pages/Shots.vue

```diff
@@ -705,7 +710,7 @@ export default {
       this.uploadShotFile((err) => {
         if (!err) {
           this.loading.importing = false
-          this.hideImportModal()
+          this.hidePreviewModal()
```

Comment:
Preview is a term use everywhere else to describe videos and pictures sent by artists. Can you look for another term like ImportRender?